provider/kubernetes/annotations_parser.go

```diff
@@ -41,21 +41,25 @@ func getSliceAnnotation(meta *v1beta1.Ingress, name string) []string {
 	return value
 }
 
-func getMapAnnotation(meta *v1beta1.Ingress, name string) map[string]string {
-	value := make(map[string]string)
-	if annotation := meta.Annotations[name]; annotation != "" {
-		for _, v := range strings.Split(annotation, ",") {
-			pair := strings.Split(v, ":")
+func getMapAnnotation(meta *v1beta1.Ingress, labelName string) map[string]string {
```

Comment:
_label_ has a different connotation in Kubernetes. We should continue to call it `name`, or go with either `annotationName` or `annotName`.

Reply:
currently I'm rewriting all labels/annotations system for all providers.
This function is extracted from this work.
I can change for now, but in the future we don't gonna separate labels and annotations.